test/controllers/AccountLinkControllerSpec.scala

```diff
@@ -180,8 +189,10 @@ class AccountLinkControllerSpec extends SpecBase {
         .thenReturn(Future.successful(None))
 
       running(app) {
-        val result = route(app, fakeRequest(
-          GET, controllers.routes.AccountLinkController.getAccountLinks(testSessionId).url)).value
+        val result = route(
+          app,
+          fakeRequest(GET, controllers.routes.AccountLinkController.getAccountLinks(testSessionId).url)
+        ).value
```

Comment:
There is a new line between this and the other line in the other running app sections, it might be worth mirroring it here for readability/consistency.

Reply:
done.